src/pages/About.js

```diff
@@ -1,9 +1,11 @@
 import React from "react";
+import '../styles/About.css'
 
 function About(props) {
   return (
     <div class="container">
     <h1>About</h1>
+    <About />
```

Comment:
Here you are trying to use the component About.js but you have not imported this file. But because of the component and page having exactly the same file name this page is trying to render itself in a endless cycle. This causes Chrome to crash. The name of the component file needs to be changed and then imported into the About page file.